### PR TITLE
Add Homebrew installation instructions

### PR DESCRIPTION
**Pull request by croaky**
Link to Clearbit the first time we mention it.

### CODE REVIEW

**Comment by bernerdschaefer** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -4,20 +4,33 @@ clearbit
...
 
-```go
-import "github.com/thoughtbot/clearbit"
+Install
+-------
+
+For OS X users, you can use the Homebrew package:
+
+```
+brew tap thoughtbot/formulae
```
Comment (bernerdschaefer) [inline README.md]: It looks like you can do `brew install thoughtbot/formulae/clearbit` instead!

Comment (croaky) [inline README.md]: Oooh, nice. For people who have already tapped `thoughtbot/formulae`, it seems like both approaches won't help them install it:

```
% brew install thoughtbot/formulae/clearbit
Error: No available formula with the name "thoughtbot/formulae/clearbit"
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
Error: No formulae found in taps.
==> You haven't updated Homebrew in a while.
A formula for thoughtbot/formulae/clearbit might have been added recently.
Run `brew update` to get the latest Homebrew updates!
```

I'm thinking maybe the instructions should be:

```
brew untap thoughtbot/formulae
brew tap thoughtbot/formulae
brew install clearbit
```

Is a Go installation also a prerequisite?

```
% brew install clearbit
==> Installing clearbit from thoughtbot/formulae
==> Downloading https://github.com/thoughtbot/clearbit/archive/v0.1.1.tar.gz
Already downloaded: /Library/Caches/Homebrew/clearbit-0.1.1.tar.gz
==> Cloning https://github.com/codegangsta/cli.git
Updating /Library/Caches/Homebrew/clearbit--github.com-codegangsta-cli--git
==> Checking out revision 839f07bfe4819fa1434fa907d0804ce6ec45a5df
==> go build -o /usr/local/Cellar/clearbit/0.1.1/bin/clearbit -v github.com/thoughtbot/clearbit/cmd/clearbit
🍺  /usr/local/Cellar/clearbit/0.1.1: 2 files, 8.8M, built in 4 seconds
```

Homebrew might install Go if it is missing?

Comment (bernerdschaefer) [inline README.md]: > For people who have already tapped thoughtbot/formulae, it seems like both approaches won't help them install it

For this case, I think you just need a normal `brew update` first, to pick up the new formula.

> Is a Go installation also a prerequisite?
> Homebrew might install Go if it is missing?

Homebrew will indeed install Go if its missing. It won't be linked `/usr/local/bin/go`, and doesn't require a working Go installation (GOPATH, etc) though.

Comment (croaky) [inline README.md]: Cool, just tried a few variations on my machine. How do the instructions look now?

**Comment by bernerdschaefer** [comment]
Thanks for this! Looks great. 👍

**Pull request merged**